Gemfile

```diff
@@ -60,6 +60,6 @@ else
   gem 'urbanopt-reopt', github: 'URBANopt/urbanopt-reopt-gem', branch: 'develop'
 end
 
-gem 'openstudio-load-flexibility-measures', github: 'NREL/openstudio-load-flexibility-measures-gem', branch: 'v0.1.3'
+gem 'openstudio-load-flexibility-measures', github: 'NREL/openstudio-load-flexibility-measures-gem', branch: 'master'
 
```

Comment:
I would prefer this to be pinned to the release instead of to `master`, so there is a bit more stability. Can you revert this change? As of right now v0.1.3 == master